redbot/core/events.py

```diff
@@ -375,13 +377,59 @@ async def on_guild_remove(guild: discord.Guild):
 
     @bot.event
     async def on_cog_add(cog: commands.Cog):
-        bot.counter.inc_raw("Red", "on_cog_add")
+        bot.counter._inc_core_raw("Red_Core", "on_cog_add")
         confs = get_latest_confs()
         for c in confs:
             uuid = c.unique_identifier
             group_data = c.custom_groups
             await bot._config.custom("CUSTOM_GROUPS", c.cog_name, uuid).set(group_data)
 
+    @bot.event
+    async def on_voice_state_update(
+        member: discord.Member, before: discord.VoiceState, after: discord.VoiceState
+    ) -> None:
+        bot.counter._inc_core_raw("Red_Core", "on_voice_state_update")
+
+    @bot.event
+    async def on_message_edit(_prior, message):
+        bot.counter._inc_core_raw("Red_Core", "on_message_edit")
+
+    @bot.event
+    async def on_user_update(before: discord.User, after: discord.User):
+        bot.counter._inc_core_raw("Red_Core", "on_user_update")
+
+    @bot.event
+    async def on_member_update(before: discord.Member, after: discord.Member):
+        bot.counter._inc_core_raw("Red_Core", "on_member_update")
+
+    @bot.event
+    async def on_member_join(member: discord.Member):
+        bot.counter._inc_core_raw("Red_Core", "on_member_join")
+
+    @bot.event
+    async def on_red_audio_track_start(guild: discord.Guild, track, requester: discord.Member):
+        bot.counter._inc_core_raw("Red_Core", "on_red_audio_track_start")
+
+    @bot.event
+    async def on_red_audio_queue_end(guild: discord.Guild, track, requester: discord.Member):
+        bot.counter._inc_core_raw("Red_Core", "on_red_audio_queue_end")
+
+    @bot.event
+    async def on_red_audio_track_end(guild: discord.Guild, track, requester: discord.Member):
+        bot.counter._inc_core_raw("Red_Core", "on_red_audio_track_end")
+
+    @bot.event
+    async def on_raw_reaction_add(payload: discord.RawReactionActionEvent):
+        bot.counter._inc_core_raw("Red_Core", "on_raw_reaction_add")
+
+    @bot.event
+    async def on_trivia_end(session):
+        bot.counter._inc_core_raw("Red_Core", "on_trivia_end")
+
+    @bot.event
+    async def on_filter_message_delete(message, hits):
+        bot.counter._inc_core_raw("Red_Core", "on_filter_message_delete")
+
```

Comment:
This Part of the PR is not final ... i just added here for the purpose of tracking the events monitored/dispatched by core and core cogs.... adding it here would make it easier to not have duplicated counts... but may not be ideal.